azurerm/internal/services/kusto/tests/kusto_cluster_resource_test.go

```diff
@@ -195,6 +195,30 @@ func TestAccAzureRMKustoCluster_vnet(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMKustoCluster_optimizedAutoScale(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_kusto_cluster", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMKustoClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMKustoCluster_optimizedAutoScale(data),
```

Comment:
since it's possible to update these values too, can we add a test disabling/updating/disabling these settings too?